src/pages/Index.js

```diff
@@ -17,38 +19,91 @@ const modalDescriptionInput = document.querySelector(
   "#modal-description-input"
 );
 
+// AVATAR
```

Comment:
Needs correction: I can't like cards, when I press on the like button Yes button appears and nothing happens